### PR TITLE
doc: Update Gatekeeper warning

### PR DESCRIPTION
**Pull request by bobsut**
After making all changes to the cask:

- [N/A] `brew cask audit --download {{cask_file}}` is error-free.
- [N/A] `brew cask style --fix {{cask_file}}` reports no offenses.
- [*] The commit message includes the cask’s name and version.
- [N/A] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [N/A] Named the cask according to the [token reference].
- [N/A] `brew cask install {{cask_file}}` worked successfully.
- [N/A] `brew cask uninstall {{cask_file}}` worked successfully.
- [*] Checked there are no [open pull requests] for the same cask.
- [N/A] Checked the cask was not [already refused].
- [*] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

This addresses Issue GH-59914.

### CODE REVIEW

File: doc/faq/the_app_cant_be_opened.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+or this:
+
+<img src="https://i.imgur.com/ouAUElQ.png" width="593" alt="Mojave Gatekeeper message">
+
 This is a [security feature from Apple](https://support.apple.com/en-us/HT202491). The single most important thing to know is that **you can allow individual apps to be exempt from that feature.** This allows the app to run while the rest of the system remains under protection.
 
 **Always leave system-wide protection enabled,** and disable it only for specific apps as needed.
 
-If you are sure you want to trust the app, you can disable protection for that app by right-clicking its icon and choosing `Open`:
+If you are sure you want to trust the app, you can disable protection for that app by right-clicking or control-clicking its icon and choosing `Open`:
```
Comment (vitorgalvao) [inline doc/faq/the_app_cant_be_opened.md]: This complicates the message needlessly.

File: doc/faq/the_app_cant_be_opened.md [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+# or
+# _The app_ is an app downloaded from the Internet. Are you sure you want to open it?
 
 Chances are that certain apps will give you a popup message like this:
 
 <img src="https://i.imgur.com/CnEEATG.png" width="532" alt="Gatekeeper message">
 
+or this:
+
+<img src="https://i.imgur.com/ouAUElQ.png" width="593" alt="Mojave Gatekeeper message">
```
Comment (vitorgalvao) [inline doc/faq/the_app_cant_be_opened.md]: As stated in the issue, this message is not exclusive to Mojave. Also, this image is a different style (Dark Mode) and lower quality, making it inconsistent.

File: README.md [hunk 3]
```diff
@@ -54,7 +54,8 @@ If the issue is still not solved, see the guides for common problems:
 * [`Checksum does not match` error](doc/reporting_bugs/checksum_does_not_match_error.md)
 * [`source is not there` error](doc/reporting_bugs/source_is_not_there_error.md)
 * [`wrong number of arguments` error](doc/reporting_bugs/wrong_number_of_arguments_error.md)
-* [The app can’t be opened because it is from an unidentified developer](doc/faq/the_app_cant_be_opened.md)
+* [_The app_ can’t be opened because it is from an unidentified developer](doc/faq/the_app_cant_be_opened.md)
```
Comment (vitorgalvao) [inline README.md]: There’s no need to italicise “The app”. It’s not not supposed to be the exact string (else it’d be between quotes or backticks, like the other errors are), it’s a guideline of the *reason*.

**Review by vitorgalvao — COMMENTED** [review]
“Downloaded from the internet” is a different message altogether. It shows even if the app is codesigned, and does not require the right click to bypass.

Thank you for the submission, but this complicates the messaging for little benefit.

File: doc/faq/the_app_cant_be_opened.md (hunk 1 above)
Comment (reitermarkus) [reply doc/faq/the_app_cant_be_opened.md]: I think the last time I control-clicked anything was when this beauty was still state-of-the-art: https://en.wikipedia.org/wiki/Apple_Pro_Mouse

**Comment by vitorgalvao** [comment]
To expand on the given reasons, here’s a screenshot of the “downloaded from the internet” message on a codesigned app, opened normally (no right-click), on Sierra.

**Pull request closed without merging**